### PR TITLE
[tests-only][full-ci]Use directory references from global dir variable in drone.star file

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description

it is better to use directory references from the dir var, rather than using the hard-coded values in every use case.

## Related Issue

- https://github.com/owncloud/QA/issues/734#issue-1173320353

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
LGTM

File: .drone.star [hunk 1]
```diff
@@ -2784,11 +2786,11 @@ def fixPermissions(phpVersion, federatedServerNeeded, selUserNeeded = False, pat
         ] if (pathOfServerUnderTest != dir["server"]) else []) + ([
             "chown -R www-data %s" % dir["federated"],
         ] if federatedServerNeeded else []) + ([
-            "chmod 777 /home/seluser/Downloads/",
+            "chmod 777 %s/" % dir["browserServiceDir"],
```
Comment (grgprarup) [inline .drone.star:2789]:
```suggestion
            "chmod 777 %s" % dir["browserServiceDir"],
```
To make consistent with lines #2113, and #2099

File: .drone.star [hunk 2]
```diff
@@ -2784,11 +2786,11 @@ def fixPermissions(phpVersion, federatedServerNeeded, selUserNeeded = False, pat
...
         ] if selUserNeeded else []),
         "volumes": [{
             "name": "downloads",
-            "path": "/home/seluser/Downloads/",
+            "path": "%s/" % dir["browserServiceDir"],
```
Comment (grgprarup) [inline .drone.star:2793]:
```suggestion
            "path": "%s" % dir["browserServiceDir"],
```

**Review by grgprarup — COMMENTED** [review]
lgtm :+1:

File: .drone.star [hunk 3]
```diff
@@ -37,6 +37,8 @@ dir = {
     "base": "/drone",
     "server": "/drone/src",
     "federated": "/drone/federated",
+    "core": "/drone/core",
```
Comment (kiranparajuli589) [inline .drone.star:40]:
```suggestion
    "testrunner": "/drone/core",
```
this is a testrunner basically.

Comment (amrita-shrestha) [reply .drone.star:40]: why?

Comment (kiranparajuli589) [reply .drone.star:40]: hmm..then sth like `tarballServer`.

**Review by kiranparajuli589 — COMMENTED** [review]
looks good :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**